frontend/src/components/pages/MultiUserBurndownChart/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import Presentation from "./Presentation";
+import { connect } from "react-redux";
+import { useParams } from "react-router-dom";
+import { fillInSnapshotDateGaps } from "./utils";
+// import { apiReduxApps } from "../../../apiAccess/apiApps";
+
+export function MultiUserBurndownChartUnconnected({
+  //   mapStateToProps
+  currentTeamBurndownStats,
+  userTeam,
+  // mapDispatchToProps
+
+  // storybook
+  forceUser,
+}) {
+  let urlParams = useParams() || {};
+  const authedUser = forceUser || parseInt(urlParams.userId);
```

Comment:
This won't work. The user id will be undefined since this will be on a team page and not a user page. 

Maybe just stop trying to highlight one specific user. I'm not sure why you want to do that anyway, it's not a requirement, it's making the code more complicated, and it wont even work in context since we are looking at a team.

This line also doesn't make sense. Are you saying that if there is a userId in the url then the current loggede in user is always that user? Is this just bad naming conventions? Or is it a misunderstanding.

If I log in and my userid is 2 then  the authenticated user has an id of 2.
If I go and look at your board and you have an id of 512, then the authenticaed user still has an id of 2.